lib/stripped-build-plugins.js

```diff
@@ -36,7 +37,26 @@ module.exports = function(environment) {
           'warn',
           'debugSeal'
         ]
-      })
+      }),
+      // comment out when running non-baseline production benchmarks
+      // WARNING do not ever commit the commented out version!
+      stripHeimdall
```

Comment:
@fivetanley I've found it very hard to replicate a production build without using the production env, if you have suggestions I'm open. Was going to look into a process flag to use as a toggle to keep us safe.